vagrantcommand.go

```diff
@@ -15,15 +15,25 @@ limitations under the License.
 
 package vagrantssh
 
+import "os"
 import "os/exec"
 
 // VagrantCommand is a command that is run on a vagrant node
 type VagrantCommand struct {
 	ContivNodes int
+	Env        string
```

Comment:
OK I see the problem now. Replace `os.Env`'s value with `VagrantCommand.Env`. This type should be `[]string`, not `string`.